cellprofiler/modules/classifyobjects.py

```diff
@@ -247,110 +254,113 @@ def add_single_measurement(self, can_delete=True):
+Select how you want to define the spacing of the bins. You have the
+following options:
+
+-  *%(BC_EVEN)s:* Choose this if you want to specify bins of equal
+   size, bounded by upper and lower limits. If you want two bins, choose
+   this option and then provide a single threshold when asked.
+-  *%(BC_CUSTOM)s:* Choose this option to create the indicated number
+   of bins at evenly spaced intervals between the low and high
+   threshold. You also have the option to create bins for objects that
+   fall below or above the low and high threshold.
+""" % globals()))
 
         group.append("bin_count", cps.Integer(
-                "Number of bins", 3, minval=1, doc="""
-            This is the number of bins that will be created between
-            the low and high threshold"""
+                "Number of bins", 3, minval=1, doc="""\
+This is the number of bins that will be created between
+the low and high threshold"""
         ))
 
         group.append("low_threshold", cps.Float(
-                "Lower threshold", 0, doc="""
-            <i>(Used only if "%(BC_EVEN)s" selected)</i><br>
-            This is the threshold that separates the lowest bin from the
-            others. The lower threshold, upper threshold, and number of bins
-            define the thresholds of bins between the lowest and highest.
-            """ % globals()))
+                "Lower threshold", 0, doc="""\
+*(Used only if "%(BC_EVEN)s" selected)*
+
+This is the threshold that separates the lowest bin from the others. The
+lower threshold, upper threshold, and number of bins define the
+thresholds of bins between the lowest and highest.
+""" % globals()))
 
         group.append("wants_low_bin", cps.Binary(
-                "Use a bin for objects below the threshold?", False, doc="""
-            Select <i>%(YES)s</i> if you want to create a bin for objects
-            whose values fall below the low threshold. Select <i>%(NO)s</i>
-            if you do not want a bin for these objects.
-            """ % globals()))
+                "Use a bin for objects below the threshold?", False, doc="""\
+Select "*%(YES)s*" if you want to create a bin for objects whose values
+fall below the low threshold. Select "*%(NO)s*" if you do not want a bin
+for these objects.
+""" % globals()))
 
         def min_upper_threshold():
             return group.low_threshold.value + np.finfo(float).eps
 
         group.append("high_threshold", cps.Float(
                 "Upper threshold", 1,
-                minval=cps.NumberConnector(min_upper_threshold), doc="""
-            <i>(Used only if "%(BC_EVEN)s" selected)</i><br>
-            This is the threshold that separates the last bin from
-            the others.
-            Note that if you would like two bins, you should select <i>%(BC_CUSTOM)s</i>.
-            """ % globals()))
+                minval=cps.NumberConnector(min_upper_threshold), doc="""\
+*(Used only if "%(BC_EVEN)s" selected)*
```

Comment:
Please change to  `*(Used only if using a single measurement and "%(BC_EVEN)s" selected)*`